src/plugins/sensor/AirSimSensor/AirSimSensor.cpp

```diff
@@ -233,8 +236,10 @@ void AirSimSensor::request_images() {
         // Get Lidar Data
         if (get_lidar_data_) {
             AirSimLidarType l;
-            l.lidar_data = img_client->getLidarData(lidar_name, vehicle_name);
```

Comment:
My recollection is that this was causing the plugin to crash, but I can't remember the details. I'll look into it.

Reply:
I had placeholder names hard coded: "Drone1", "Lidar1". These match the vehicle/sensor names in the settings.json file on the windows side. If they don't match then it will crash.